src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,157 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_USERS_AGE = 18;
+    private static final int MAX_USERS_AGE = 150;
+    private static final int CORRECT_USERS_AGE = 25;
+    private static final int UNDER_MIN_USERS_AGE = 16;
+    private static final int OVER_MAX_USERS_AGE = 190;
+    private static final int FIRST_INCORRECT_AGE = 3965496;
+    private static final int SECOND_INCORRECT_AGE = -32;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        User arnold = new User();
+        arnold.setAge(CORRECT_USERS_AGE);
+        arnold.setLogin("Arny");
+        arnold.setPassword("1*hD#Sak");
+        storageDao.add(arnold);
+    }

```

Comment:
it is not necessary to perform this action before each test, 
as far as I understand it is needed only for this test "register_sameLogin_notOk()", 
then you only need to write it there